### ISSUE TITLE
No defaultAction for GenericTemplate

### ISSUE THREAD

**Issue by Iseldore** [CasperLaiTW/laravel-fb-messenger#80]
Hi again,

With `ListTemplate` we can add a defaultAction when the element is clicked, like your example : 

```
$defaultButton = new UrlButton(null, 'https://developers.facebook.com/docs/messenger-platform/send-api-reference/list-template');
$template->addElement('Moving list', 'Help you to check all of them', 'http://www.mk-sokolovi.com/wp/wp-content/uploads/2013/07/T-shirt-Vector.jpg')
        ->setDefaultAction($defaultButton)
        ->buttons()
        ->addWebButton('View', 'https://developers.facebook.com/docs/messenger-platform/send-api-reference/list-template');
```

but we can't do the same thing with `GenericTemplate` while it is mentioned in the Messenger's docs : https://developers.facebook.com/docs/messenger-platform/send-api-reference/generic-template

It throws an exception : `Call to undefined method Casperlaitw\LaravelFbMessenger\Messages\Element::setDefaultAction()`

**Comment by CasperLaiTW** [comment]
Cool. Thank you catch.

I will implement.

**Comment by CasperLaiTW** [comment]
Hi, I released new version 1.4.7

Enjoy it :)
